tamr_unify_client/client.py

```diff
@@ -1,3 +1,5 @@
+from urllib.parse import urljoin
+
```

Comment:
nit: no newlines between imports from 3rd party libs please!

Reply:
IntelliJ / Pycharm REALLY wants them in there, but I will fix.